docs/getting-started/overlay.rst

```diff
@@ -195,6 +195,7 @@ Put the required configuration into ``config.py``.
 
     PACKAGES = [
         "myoverlay.base",
+        "invenio.base",
```

Comment:
Good catch! Can you move it after `invenio.modules.*` (see https://github.com/inveniosoftware/invenio/blob/master/invenio/base/config.py#L61-L62)? Thanks